tls/src/keys.rs

```diff
@@ -384,4 +384,5 @@ impl RotCertVerifier {
 pub struct SprocketsConfig {
     pub resolve: ResolveSetting,
     pub roots: Vec<Utf8PathBuf>,
+    //pub corpus: Vec<Utf8PathBuf>,
```

Comment:
Noting this here for reference: Because the corpus is stored on the M.2 we can't know the corpus path before boot. This config currently lives as part of the smf .toml file so trying to do a mixture of static + runtime was messy, hence the passing in the corpus at runtime to connect/accept.